sys/make.x86_64-linux-gnu

```diff
@@ -3,6 +3,10 @@
 # gfortran/gcc 5.4 - 7.1
 ############################################################################
 
+# Directory where standard packages are installed in the system
+PREFIX := /usr/local
```

Comment:
Not sure if this should be /usr/ instead if its a 'standard' install ( or even /opt/ ).

Reply:
I think `/usr` is the correct choice if things were installed as packages. I changed this in all 4 makefiles.